src/DependencyInjection/Configuration.php

```diff
@@ -98,6 +98,15 @@ public function getConfigTreeBuilder(): TreeBuilder
                         ->end()
                     ->end()
                 ->end()
+                ->arrayNode('metadata')
+                    ->addDefaultsIfNotSet()
+                    ->children()
+                        ->booleanNode('cache')
+                            ->info('Whether or not to cache the metadata')
+                            ->defaultValue('%kernel.debug%')
```

Comment:
It should be `%kernel.debug%` negated, so when `%kernel.debug%` is `false`, the cache is enabled

Reply:
wow, right XD another coffee needed :dancer: